tmail-backend/jmap/extensions-api/src/main/java/com/linagora/tmail/james/jmap/EventAttendanceRepository.java

```diff
@@ -19,21 +19,16 @@
 package com.linagora.tmail.james.jmap;
 
 import java.util.List;
-import java.util.Optional;
 
 import org.apache.james.core.Username;
 import org.apache.james.jmap.mail.BlobId;
 import org.reactivestreams.Publisher;
 
 import com.linagora.tmail.james.jmap.model.CalendarEventAttendanceResults;
-import com.linagora.tmail.james.jmap.model.CalendarEventReplyResults;
-import com.linagora.tmail.james.jmap.model.LanguageLocation;
 
 public interface EventAttendanceRepository {
 
    Publisher<CalendarEventAttendanceResults> getAttendanceStatus(Username username, List<BlobId> blobIds);
    
-   Publisher<CalendarEventReplyResults> setAttendanceStatus(Username username, AttendanceStatus attendanceStatus,
-                                                            List<BlobId> eventBlobIds,
-                                                            Optional<LanguageLocation> maybePreferredLanguage);
+   Publisher<Void> setAttendanceStatus(Username username, AttendanceStatus attendanceStatus, BlobId eventBlobId);
```

Comment:
The `xxxRepository` should follow its responsibility as implied by its name. Embedding JMAP Result POJO (view layer) here is not good pattern

Adding the argument `Optional<LanguageLocation> maybePreferredLanguage` feels like a hack to pass data for only Standalone class to generate reply emails. This does not belong to this API.